Add phone calls & available models (#26)
fixie-ai/ultradox: docs/src/content/docs/availablemodels.mdx

---
  title: "Available Models"
---

The Ultravox API currently provides the following models.

<table class="w-full">
    <tr class="w-full">
        <th class="w-4/12">Model</th>
        <th class="w-8/12">Description</th>
    </tr>
    <tr>
        <td class="font-mono">fixie-ai/ultravox</td>

>         <td>This is an alias that points to `ultravox-70B`. The default if no model is specified when creating a call with the API.</td>

Reviewer: Ideally we'd word this such that we don't have to update docs every time we change this.  Maybe something like "This is an alias for our most sophisticated model."

Author: yeah makes sense. gonna keep for now and revisit when we have more models and it's more complicated/harder to update.

    </tr>
    <tr>
        <td class="font-mono">fixie-ai/ultravox-70B</td>
        <td>70B variant of Ultravox. Supports tools.</td>
    </tr>
    <tr>
        <td class="font-mono">fixie-ai/ultravox-8B</td>

>         <td>8B variant of Ultravox. Not recommended for most use cases. Does not support tools.</td>

Reviewer: ```suggestion
        <td>8B variant of Ultravox. Not recommended for most use cases. Tools are unlikely to work.</td>
```

(I don't love this, but it's more accurate.  Feel free to word smith more.)
